CONTRIBUTING.md

```diff
@@ -24,24 +24,35 @@ only use squashing if the resulting squash commit can accuratly describe the cha
 Once the change is pushed to the main branch, it is time for a release.
 
 ## Releases & Versioning
-Releases happen from a separate branch, called `release`. This is because lerna eagarly publishes
-whenever files in a package have changed. Tests and markdown files are ignored, but all other changes will trigger
-at minimum a PATCH bumb.
-
-**Always** push the HEAD of the main branch to `release`.
-This will trigger a deploy on travis CI.
-Here lerna will look at changes since the last release (by using git tags),
-and find out what changes have occured, and what commit messages affect each package.
-Eventual version bumps will be made to `package.json` files automatically,
-and commit descriptions will be appended to changelogs.
-The changes are then committed back to the `release` branch and tagged.
-Packages are published to npm, and a github release is made.
-A pull request is opened back to the main branch.
-It is important that the commit is pushed with the same hash,
-to keep git tags correct.
-
-Documentation is also built and deployed from the release branch.
-If the code snippets have changed, a pr is made to the service contracts repo.
+Releases are done from the main branch, so when a pull request is merged,
+CI/CD will run tests, and if successful, do deploys.
+Documentation is built and deployed, and code snippets
+are exported to the service contract repo as a pull request.
+
+Updating and uploading npm packages only happens if the HEAD commit of the main branch
+contains `[release]` in its description. When CI/CD sees this, it will use lerna to update
+package versions of changed packages based on commit messages, and add the 
+changes to the changelogs. The changes are comitted to the main branch
+with the new versions as git tags, and the new package versions are uploaded to npm.
+
+We restrict new npm releases to `[release]`-tagged commits becuase lerna is
+quite agressive in its versioning. Changes to any file not ignored by lerna will
+cause a PATCH bump. Markdown files and tests are ignored, but chaning anything else,
+like a comment in a source file, will trigger a new version,
+irrespective of conventional commits.
+
+This does *not* mean you should store unfinished work on the main branch.
+Another package may be ready for release, and once anyone pushes a `[release]`
+commit, all changed packages are updated.
+
+Also keep in mind that the `[release]` commit has to be the HEAD of
```

Comment:
maybe we just advice to trigger the push only for administrators?
so it won't be as a part of pr but rather the direct push

Reply:
we have now a bit of a problem now with people merging to master without us knowing it